plugins/transport/socket/main.go

```diff
@@ -31,7 +32,9 @@ func rate() int64 {
 }
 
 type configT struct {
-	Path         string `validate:"required"`
+	Path         string
+	Type         string
+	Url          string
```

Comment:
Thanks to the bot for bringing it up. "127.0.0.1:3060" is not a url since it's missing a scheme and doesn't support a path. "socketaddr" may be more appropriate

Reply:
Agree. Then you don't need two config parameters.

Reply:
Yes, the socketaddr sounds a lot better. I'm not sure about not needing 2 config parameters. I think it would be a good idea for backwards compatibility of the config file to leave the path as a possible parameter for unix sockets.

Reply:
Yes please, otherwise we need to adjust SGO to account for any configuration file changes.